### PR TITLE
add merchant and test class

### PR DESCRIPTION
**Pull request by carynkeffer**
Add merchant and test class.

### CODE REVIEW

**Review by tsnieuwen — APPROVED** [review]
Looks good to me! (Granted I was the navigator...)

**Comment by Omegaeye** [comment]
looks good to me

File: test/merchant_test.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/merchant'
+
+class MerchantTest < Minitest::Test
+  def test_it_exists
```
Comment (b-enji-cmd) [inline test/merchant_test.rb:6]: You can organize a `def setup` at the top of a test file so you can move 
```
@m = Merchant.new({
                       :id => 5,
                       :name => "Turing School"
                       })
```

Makes things a little easier so you don't have to keep pasting a bunch

**Review by b-enji-cmd — APPROVED** [review]
Good work!

**Review by Omegaeye — APPROVED** [review]
looks good to me

**Pull request merged**